lab_1/start.py

```diff
@@ -15,11 +16,30 @@
     with open(os.path.join(PATH_TO_TEXTS_FOLDER, 'de.txt'), 'r', encoding='utf-8') as file_to_read:
         de_text = file_to_read.read()
 
+    with open(os.path.join(PATH_TO_TEXTS_FOLDER, 'la.txt'), 'r', encoding='utf-8') as file_to_read:
+        la_text = file_to_read.read()
+
     with open(os.path.join(PATH_TO_TEXTS_FOLDER, 'unknown.txt'), 'r', encoding='utf-8') as \
             file_to_read:
         unknown_text = file_to_read.read()
 
+    lang_unk = main.create_language_profile('unk', unknown_text, [])
+    lang_en = main.create_language_profile('en', en_text, [])
+    lang_de = main.create_language_profile('de', de_text, [])
+    lang_la = main.create_language_profile('la', la_text, [])
+
+    TOP_N = 4
+
+    langs = [lang_en, lang_de, lang_la]
+
+    det = main.detect_language(lang_unk, lang_en, lang_de, TOP_N)
+
+    det_adv = main.detect_language_advanced(lang_unk, langs, [lang_la, lang_de], TOP_N)
+
     EXPECTED = 'en'
-    RESULT = ''
+    RESULT = det
+    print('basic result is', RESULT)
+    RESULT_ADV = det_adv
+    print('advanced result is',RESULT_ADV)
```

Comment:
I'm curious, what are these results?

Reply:
RESULT is the result of detect_language function and RESULT_ADV is the result of the detect_language_advanced function (and I've already rewritten this as now there is no RESULT_ADV as it is now just RESULT, because I think it is more needed whether my target score is 8) (also now RESULT takes function itself with no other variables needed)
Results were en for the RESULT and None for the RESULT_ADV as I accidentally wrote not names of languages for the languages names but profiles, so my whole function wasn't working. But now everything is ok, I guess.